tests/test_default_extraction_patterns.py

```
@@ -0,0 +1,108 @@
+            "mt-esp32c6-ota.bin",
+        ],
+    )
+    def test_mt_prefix_alone_matches_all_esp32_ota_variants(self, filename):
+        # Single `mt-` pattern is sufficient for every ESP32 variant.
+        assert matches_extract_patterns(filename, ["mt-"])
+
+    @pytest.mark.parametrize(
+        [
+            "filename",
+            "should_match",
+        ],
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)